js/pulse.js

```diff
@@ -106,7 +106,6 @@ var newProjectForm = {
     newProjectForm.formContainer.style.display = 'none';
     newProjectForm.projectContainer.style.display = 'block';
     newProjectForm.toggleFormButton.style.transform = 'rotate(0deg)';
-    setTimeout(function(){ PulseMaker.getData(); }, 250);
```

Comment:
Can we update the comment here
>  this turns on both auto app refresh and browser notification

 https://github.com/mmmavis/network-pulse/blob/9c2ed73f25c92a0a2d10d281d617d35bbe07f465/js/pulse.js#L7

To only mention browser notifications?